pom.xml

```diff
@@ -198,7 +198,7 @@
   </pluginRepositories>
 
   <scm>
-    <connection>scm:git:ssh://git@github.com:chrisdennis/terracotta-parent.git</connection>
-    <developerConnection>scm:git:ssh://git@github.com:chrisdennis/terracotta-parent.git</developerConnection>
+    <connection>scm:git:ssh://git@github.com:Terracotta-OSS/terracotta-parent.git</connection>
```

Comment:
Connection should point to the read-only https https://github.com/Terracotta-OSS/terracotta-parent.git